fix: docs/framework/wcf/feature-details/data-contract-schema-reference.md

nschonni opened the pull request:
- xlms split across lines
- trim trialing/leaading space
- spacing around headings

rpetrusha commented on docs/framework/wcf/feature-details/data-contract-schema-reference.md:
````diff
@@ -1,680 +1,691 @@
+
+```xml
+<xs:simpleType name="MyEnum">
+<xs:restriction base="xs:string">
````
Levels of indentation here should be fixed. \<xs:restriction> is a child of \<xs:simpletype>.

nschonni replied:
OK, I left that as-is, but i'll go fix that now

nschonni replied:
Ran an XML formatter on this block. Left the other ones since it made it a little noisy by pulling attributes to the same line

rpetrusha reviewed (changes requested):
I've left one comment for you to consider, @nschonni.

nschonni committed:
fix: docs/framework/wcf/feature-details/data-contract-schema-reference.md

- xlms split across lines
- trim trialing/leaading space
- spacing around headings
- format xml block

rpetrusha commented:
Thanks for correcting the formatting of the XML block, @nschonni. I'll merge your PR now.
